_search-plugins/knn/knn-score-script.md

```diff
@@ -323,6 +323,11 @@ A space corresponds to the function used to measure the distance between two poi
     <td>\[ d(\mathbf{x}, \mathbf{y}) = \text{countSetBits}(\mathbf{x} \oplus \mathbf{y})\]</td>
     <td>\[ score = {1 \over 1 + d } \]</td>
   </tr>
+  <tr>
```

Comment:
@junqiu-lei This row duplicates the previous row (which says that the space type is `hammingbit`). Could you take a look?

Reply:
Never mind, just saw the comment from Heemin.